task_list.md

```
@@ -0,0 +1,23 @@
+# Task List — work-5c1ec819
+
+## Implementation Tasks
+
+- [ ] 1. Create `scripts/check_release_history.sh` shell script
+  - [ ] Enumerate v* tags (excluding v*-rc* prerelease patterns)
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)